organizational/conduct.rst

```diff
@@ -0,0 +1,101 @@
+The Mission of Hacker Society
+=============================
+
+A time and place for people to learn about technology from each other, in order
+to build a community, to empower the individual, promote equality, and build an
```

Comment:
Looks better, but there's still an issue with the infinitives. See the all caps sections of @mrb113's comment above